<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coursemanage</class>
 <widget class="QWidget" name="coursemanage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>课程管理</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>课程号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="course_IDlineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="course_querypushButton">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>课程名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>授课老师：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>授课时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>授课地点：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>学时：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>学分：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="couresnamelineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="course_teacherlineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="course_timelineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="course_addresslineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="periodlineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="creditlineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="course_addpushButton">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="course_deletepushButton">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="course_updatepushButton">
       <property name="text">
        <string>更改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="course_backpushButton">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
